Update README.md

dchai76 opened the pull request:
Minor README formatting changes

dchai76 committed:
Update README.md

Minor README formatting changes

smusali commented on README.md:
````diff
@@ -1,41 +1,41 @@
 ```
-    $git clone https://github.com/logdna/logdna-cloudfoundry.git
-    $cd firehose-to-syslog
+    git clone https://github.com/logdna/logdna-cloudfoundry.git
````
are these 4 trailing spaces needed in front of commands?

dchai76 replied:
I think you mean leading spaces, and I don't know. Let me follow the same format as https://github.com/cloudfoundry-community/firehose-to-syslog/blob/develop/README.md.

smusali replied:
this looks better

smusali commented on README.md:
```diff
@@ -1,41 +1,41 @@
     cf set-env firehose-to-logdna API_ENDPOINT https://api.[your cf system domain]
     cf set-env firehose-to-logdna DOPPLER_ENDPOINT wss://doppler.[your cf system domain]:443
-    cf set-env firehose-to-logdna INGESTION_KEY [your ingestion key in LogDNA]
+    cf set-env firehose-to-logdna INGESTION_KEY [your LogDNA ingestion key]
```
I'd suggest `LogDNA Ingestion Key` instead of `your LogDNA ingestion key`

dchai76 replied:
Can we leave the "your"? It's not a universal key, it's different for each customer and I think "your" makes that clearer.

smusali replied:
Yup - no harm.

smusali reviewed (changes requested):
Just few comments

dchai76 committed:
Update README.md

Make formatting consistent with https://github.com/cloudfoundry-community/firehose-to-syslog/blob/develop/README.md (from which this is forked
